docs/env.md

```diff
@@ -0,0 +1,95 @@
+# Environment variables
```

Comment:
I think it's better to use a table for each section, i.e.

## GitLab

|Name|Default value|Description|
|---|---|---|
|GITLAB_CLIENT_ID|   |   |
|GITLAB_SECRET_KEY|   |   |
|GITLAB_URL|https://gitlab.com|   |
|GITLAB_WEBHOOK_URL|   |   |
|GITLAB_WEBHOOK_SECRET|   |   |